site/assets/scss/_component-examples.scss

```diff
@@ -220,17 +220,18 @@
 
 // scss-docs-start custom-tooltip
 .custom-tooltip {
-  --bs-tooltip-bg: var(--bs-primary);
+  --bs-tooltip-bg: var(--bd-violet-bg);
```

Comment:
**[Bootstrap side]** It feels weird to have a reference to `bd-` inside our doc.